spec/lib/pdf_filler/nj1040_pdf_spec.rb

```diff
@@ -1390,7 +1463,7 @@
       )
       }
 
-      it "writes rounded tax amount $7,615.10 based on income $200,000 with 2,000 exemptions 15,000 property tax deduction and 0.0637 tax rate minus 4,042.50 subtraction" do
+      it "writes rounded tax amount $7,519.00 based on income $200,000 with 3,500 exemptions 15,000 property tax deduction and 0.0637 tax rate minus 4,042.50 subtraction" do
```

Comment:
[dust] this will cause a merge conflict / test failure with my new Lines 45,49,50,54 commit based on whoever merges first, because that also uses this number. That's fine, just a flag for whoever merges second

Reply:
sorry! just merged.

this does bring up a good point though. i had to redo these numbers which was not all that tedious, just bringing out a manual calculator for a couple minutes to make sure the new test output is correct... but it feels maybe dangerously error prone? like if i had just accepted what the test was outputting and updated the expectation, that would defeat the purpose... i don't have any ideas on how to make this easier though